docs/custom_installs/local_rtd_vm.rst

```diff
@@ -7,23 +7,23 @@ Assumptions and Prerequisites
 * Debian VM provisioned with python 2.7.x
 * All python dependencies and setup tools are installed ::
 
-  $ sudo apt-get install python-setuptools
-  $ sudo apt-get install build-essential
-  $ sudo apt-get install python-dev
-  $ sudo apt-get install libevent-dev
-  $ sudo easy_install pip 
+  sudo apt-get install python-setuptools
```

Comment:
This block is throwing a linting error and isn't formatting correctly. It gets hidden because of other unrelated test failures in master right now (grrr....) but there's definitely an issue here. Here's the link:
https://travis-ci.org/rtfd/readthedocs.org/jobs/435828043

Reply:
That's no good! I just pushed an update and we'll see if that works, thanks for letting me know!